docs/source/libraries/xgboost.rst

```diff
@@ -37,8 +38,13 @@ of an ensemble. Each node of the tree has an output score, and
 contribution of a feature on the decision path is how much the score changes
 from parent to child.
 
+.. note::
+    When explaining Booster_ predictions,
+    do not pass an ``xgboost.DMatrix`` object as ``doc``, pass a numpy array
+    or a sparse matrix instead (or have ``vec`` return them).
```

Comment:
Is this limitation just a feature which is not implemented, or is there a deeper reason DMatrix is not supported?